pkg/metrics/metrics.jsx

```diff
@@ -842,25 +843,62 @@ class CurrentMetrics extends React.Component {
     }
 }
 
-const SvgGraph = ({ data, resource, have_sat }) => {
+const SvgGraph = ({ minute, data, resource, have_sat }) => {
+    return (
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox={ "0 0 2 " + SVG_YMAX } preserveAspectRatio="none">
+            <Polygons data={data} type="use" resource={resource} have_sat={have_sat} />
+            {have_sat && <Polygons data={data} type="sat" resource={resource} have_sat />}
+        </svg>
+    );
+};
+
+const Polygons = ({ data, type, resource, have_sat }) => {
+    const transform = type == "sat" ? "1,0,0,-1,1," : have_sat ? "-1,0,0,-1,1," : "-2,0,0,-1,2,";
+    const key = type + "_" + resource;
+    const dataFiltered = data.filter(d => d && typeof d[key] === 'number');
+
+    if (!dataFiltered.length)
+        return null;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18461-20230309-121036-88868592-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)